src/constants.iced

```diff
@@ -44,6 +44,9 @@ exports.constants = constants =
       kbfs : "team.kbfs"
       settings : "team.settings"
       bot_settings : "team.bot_settings"
+    wot :
+      attest : "wot.attest"
+      react  : "wot.react"
```

Comment:
what is this? maybe comment both types?

Reply:
also, does attest_with_revoke need to be here?

Reply:
sorry, from next PR, but this got stuck in 

no, the analogy is to cryptocurrency and cryptocurrency_with_revoke above, so that we're ok to know at the sig2 outer link level if stubbing is OK